components/sysutils/net-snmp/Makefile

```diff
@@ -20,51 +20,50 @@
 #
 
 #
-# Copyright (c) 2011, 2016, Oracle and/or its affiliates. All rights reserved.
-#
-# net-snmp has a "special" build where only 32-bit items are installed into
-# proto area; so prefer 32-bit to preserve existing sourcing.
+# Copyright (c) 2011, 2019, Oracle and/or its affiliates. All rights reserved.
+# Copyright 2019 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved. 
 #
+BUILD_BITS= 64_and_32
 
-#
-# Copyright 2016 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved.
-#
+# The Makefile of this component does not support multiple build jobs.
+COMPONENT_MAKE_JOBS=1
+DOXYGEN= /usr/bin/doxygen
 
 include ../../../make-rules/shared-macros.mk
 
-DOXYGEN= /usr/bin/doxygen
 PATH=$(VPATH):$(PATH.gnu)
 
 COMPONENT_NAME=		net-snmp
-COMPONENT_VERSION=	5.7.3
-COMPONENT_REVISION=	4
-COMPONENT_FMRI=	system/management/snmp/net-snmp
+COMPONENT_VERSION=	5.8
 COMPONENT_PROJECT_URL=	http://www.net-snmp.org/
+COMPONENT_FMRI= system/management/snmp/net-snmp
 COMPONENT_SRC=    $(COMPONENT_NAME)-$(COMPONENT_VERSION)
 COMPONENT_ARCHIVE= $(COMPONENT_SRC).tar.gz
 COMPONENT_ARCHIVE_HASH= \
-    sha256:12ef89613c7707dc96d13335f153c1921efc9d61d3708ef09f3fc4a7014fb4f0
-COMPONENT_ARCHIVE_URL=	http://ftp.ntua.gr/mirror/net-snmp/net-snmp/$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
+    sha256:b2fc3500840ebe532734c4786b0da4ef0a5f67e51ef4c86b3345d697e4976adf
+COMPONENT_ARCHIVE_URL=	https://sourceforge.net/projects/net-snmp/files/net-snmp/$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
 COMPONENT_CLASSIFICATION= System/Enterprise Management
 COMPONENT_SUMMARY= Net-SNMP Agent files and libraries
+COMPONENT_BUGDB=	service/net-snmp utility/net-snmp library/net-snmp
+COMPONENT_ANITYA_ID=	2062
+
+TPNO=			47655
 
+# obsolete - reminder for common targets build/install/test
 BUILD_32_and_64 += $(BUILD_DIR_32)/.docs
+INSTALL_TARGET= build $(INSTALL_32_and_64)
+TEST_TARGET= $(TEST_32_and_64)
 
-COMPONENT_BUILD_ARGS=
```

Comment:
Does it build fine in parallel now? (COMPONENT_BUILD_ARGS=-j4 in build environment)

Reply:
will check it

Reply:
No chance. The build fails with multiple includes of some libraries.

Reply:
This means `COMPONENT_BUILD_ARGS=` should be preserved.